src/Microsoft.Health.Fhir.Core/Features/Persistence/ITransactionHandler.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Transactions;
+
+namespace Microsoft.Health.Fhir.Core.Features.Persistence
+{
+    public interface ITransactionHandler : IDisposable
+    {
+        TransactionScope BeginTransaction();
```

Comment:
We might want to think about abstracting the `TransactionScope` here to something else. It seems like an implementation detail that could differ between datastores.

Reply:
Updated.